services/web/.env

```diff
@@ -1,18 +1,29 @@
 # Note that changing the variables here will require a restart of the app server
 
-ENV_NAME=development
+##########################################################
+# Variables prefixec with SERVER or HTTP are not exposed #
```

Comment:
`prefixed` ... also do we need the MASSIVE COMMENT BLOCKS?? (I'm ok if you like it though) :)